src/cpp/rtps/builtin/discovery/participant/PDP.cpp

```diff
@@ -305,6 +306,9 @@ void PDP::initializeParticipantProxyData(
     participant_data->m_leaseDuration = attributes.builtin.discovery_config.leaseDuration;
     //set_VendorId_eProsima(participant_data->m_VendorId);
     participant_data->m_VendorId = c_VendorId_eProsima;
+    participant_data->product_version.major = FASTDDS_VERSION_MAJOR;
+    participant_data->product_version.minor = FASTDDS_VERSION_MINOR;
+    participant_data->product_version.patch = FASTDDS_VERSION_MICRO;
```

Comment:
Where is the tweak value set?

Reply:
We decided not to send that information.

Reply:
Ok, I find it a bit weird because the whole product version is being sent (even if you don't set the tweak value), but I guess you would need to set a 1-byte padding anyways if you only included the first three values in the proxy data.

Reply:
P.D. I yet don't understand what is the tweak value useful for.